src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -48,14 +50,16 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.contrib.java.lang.system.ExpectedSystemExit;
```

Comment:
can we remove https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L292 dependency now ?

Reply:
Not yet. The dependency
```
    <!-- till https://github.com/checkstyle/checkstyle/issues/7368 -->
    <dependency>
      <groupId>com.github.stefanbirkner</groupId>
      <artifactId>system-rules</artifactId>
      <version>1.19.0</version>
      <scope>test</scope>
    </dependency>
```
is used in the Powermock tests.
Junit team is working on an issue to allow Powermock to replace the classloader.